lib/chromic_pdf.ex

```diff
@@ -279,28 +287,29 @@ defmodule ChromicPDF do
 
   * `[:chromic_pdf, :join_pdfs, :start | :stop | exception]`
 
-  ## How it works
-
-  ### PDF Printing
-
-  * ChromicPDF spawns an instance of Chromium/Chrome (an OS process) and connects to its
-    "DevTools" channel via file descriptors.
-  * The Chrome process is supervised and the connected processes will automatically recover if it
-    crashes.
-  * A number of "targets" in Chrome are spawned, 1 per worker process in the `SessionPool`. By
-    default, ChromicPDF will spawn each session in a new browser context (i.e., a profile).
-  * When a PDF print is requested, a session will instruct its assigned "target" to navigate to
-    the given URL, then wait until it receives a "frameStoppedLoading" event, and proceed to call
-    the `printToPDF` function.
-  * The printed PDF will be sent to the session as Base64 encoded chunks.
-
-  ### PDF/A Conversion
-
-  * To convert a PDF to a PDF/A-3, ChromicPDF uses the [ghostscript](https://ghostscript.com/)
-    utility.
-  * Since it is required to embed a color scheme into PDF/A files, ChromicPDF ships with a copy
-    of the royalty-free [`eciRGB_V2`](http://www.eci.org/) scheme by the European Color
-    Initiative. If you need to be able to use a different color scheme, please open an issue.
+  ## On Accessibility / PDF/UA
+
+  Since its version 85, Chrome generates "Tagged PDF" files [by
+  default](https://blog.chromium.org/2020/07/using-chrome-to-generate-more.html).  These files
+  contain structural information about the document, i.e. type information about the nodes
+  (headings, paragraph, etc.), as well as metadata like node attributes (e.g., image alt texts).
+  This information allows assistive tools like screen readers to do their job. `pdfinfo`
+  identifiers these files as `Tagged: yes`, and you may review some of the contained information
+  with the `pdfinfo -struct-text <file>` command.
```

Comment:
This is an extreme example but regularly we have to generate a massive(200+ page) pdf consisting of mostly tables. When tagged the resulting PDF weighs 40MiB, while the untagged version is just 3.1MiB. 
I'm not sure if this is something that's worth mentioning in the docs since it's an extreme case that already causes chromium to take minutes generating the document, and we really shouldn't be using PDFs.

Reply:
mentioned in c48ba6db3bd91e148657b68cacd804aee072ceb8

Is it actually still possible to tell Chrome *not* to include these annotations? i.e. `--no-export-tagged-pdfs`?

Reply:
`--disable-pdf-tagging`